indico/modules/rb/client/js/common/bookings/BookingDetails.module.scss

```diff
@@ -54,3 +54,7 @@
 .rejection-form {
     width: 350px;
 }
+
+.accept-with-message {
+    border-left: 1px solid darken($color: $green, $amount: 5%) !important;
```

Comment:
in other places we use the shorter way of calling scss functions: `darken($green, 5%)`